actionview/test/abstract_unit.rb

```diff
@@ -31,6 +31,7 @@ class << self
     def env
       @_env ||= ActiveSupport::StringInquirer.new(ENV["RAILS_ENV"] || ENV["RACK_ENV"] || "test")
     end
+    def root; end;
```

Comment:
@kaspth Because of this tests failed in `CachedViewRenderTest` with following error
````bash
CachedViewRenderTest#test_render_sub_template_with_errors [/Users/ronakjangir/projects/open_source/rails/actionview/test/template/render_test.rb:234]:
--- expected
+++ actual
@@ -1 +1 @@
-"Trace of template inclusion: /Users/ronakjangir/projects/open_source/rails/actionview/test/fixtures/test/sub_template_raise.html.erb"
+"Trace of template inclusion: Users/ronakjangir/projects/open_source/rails/actionview/test/fixtures/test/sub_template_raise.html.erb"
````
But we need this to stub this method in `test/template/log_subscriber_test.rb`. What I did miss here?